domain/src/test/java/org/oppia/android/domain/exploration/ExplorationProgressControllerTest.kt

```diff
@@ -209,6 +221,77 @@ class ExplorationProgressControllerTest {
     assertThat(ephemeralState.state.name).isEqualTo("Continue")
   }
 
+  @Test
+  fun testEphemeralState_startExploration_shouldIndicateButtonAnimation() {
+    oppiaClock.setFakeTimeMode(FakeOppiaClock.FakeTimeMode.MODE_FIXED_FAKE_TIME)
+    val currentTime = oppiaClock.getCurrentTimeMs()
+    oppiaClock.setCurrentTimeMs(currentTime)
+    startPlayingNewExploration(TEST_TOPIC_ID_0, TEST_STORY_ID_0, TEST_EXPLORATION_ID_2)
+    val ephemeralState = waitForGetCurrentStateSuccessfulLoad()
+    assertThat(ephemeralState.showContinueButtonAnimation).isEqualTo(true)
+    assertThat(ephemeralState.continueButtonAnimationTimestampMs).isEqualTo(
+      currentTime + TimeUnit.SECONDS.toMillis(
+        45
+      )
+    )
```

Comment:
```suggestion
    assertThat(ephemeralState.continueButtonAnimationTimestampMs)
      .isEqualTo(currentTime + TimeUnit.SECONDS.toMillis(45))
```

Not sure what happened here, but the formatting got a bit wonky (plus the equals check can be on its own line).

Ditto elsewhere below.

Reply:
done